app/assets/v2/js/pages/bounty_detail/cosmos_extension.js

```diff
@@ -0,0 +1,83 @@
+const payWithCosmosExtension = async(fulfillment_id, to_address, vm, modal) => {
+  const amount = vm.fulfillment_context.amount * 10 ** vm.decimals;
+  const token_name = vm.bounty.token_name;
+  const tenant = vm.getTenant(token_name, vm.bounty.web3_type);
+  const chainId = 'cosmoshub-4';
+  
+  if (!window.keplr) {
+    _alert({ message: gettext('Please download or enable Keplr browser wallet extension.') }, 'danger');
+  }
+
+  try {
+    await window.keplr.enable(chainId);
+  } catch (e) {
+    _alert({ message: gettext('Please unlock Keplr browser wallet extension.') }, 'danger');
+    return;
+  }
+
+  let client = await window.CosmWasmJS.setupWebKeplr({
```

Comment:
Looking great @chibie ! Do you by chance have a link to the JS library/documentation. I can't seem to track it down

Reply:
https://cosmwasm.github.io/CosmWasmJS/